Formula/caffe.rb

```diff
@@ -19,9 +19,9 @@ class Caffe < Formula
   depends_on "protobuf"
   depends_on "szip"
   depends_on "leveldb" => :optional
+  depends_on "snappy" if build.with?("leveldb")
```

Comment:
This looks weird but it is the correct order.
https://github.com/Homebrew/brew/blob/master/Library/Homebrew/rubocops/dependency_order_cop.rb#L50